commons/src/test/java/org/eclipse/kapua/commons/event/ServiceEventClientConfigurationTest.java

```diff
@@ -0,0 +1,91 @@
+        assertNull("not_null", serviceEventClientConfiguration.getClientName());
+        assertNull("not_null", serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorAddressNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration(null , "subscriberName", serviceEventBusListener);
+        assertNull("not_null", serviceEventClientConfiguration.getAddress());
+        assertEquals("not_equals", "subscriberName", serviceEventClientConfiguration.getClientName());
+        assertEquals("not_equals", serviceEventBusListener, serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorClientNameNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration("address", null, serviceEventBusListener);
+        assertEquals("not_equals", "address", serviceEventClientConfiguration.getAddress());
+        assertNull("not_null", serviceEventClientConfiguration.getClientName());
+        assertEquals("not_equals", serviceEventBusListener, serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorServiceEventNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration("address", "subscriberName", null);
+        assertEquals("not_equals", "address", serviceEventClientConfiguration.getAddress());
+        assertEquals("not_equals", "subscriberName", serviceEventClientConfiguration.getClientName());
+        assertNull("not_null", serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorAddressAndNameNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration(null, null, serviceEventBusListener);
+        assertNull("not_null", serviceEventClientConfiguration.getAddress());
+        assertNull("not_null", serviceEventClientConfiguration.getClientName());
+        assertEquals("not_equals", serviceEventBusListener, serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorNameAndServiceEventNullTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration("address", null, null);
+        assertEquals("not_equals", "address", serviceEventClientConfiguration.getAddress());
+        assertNull("not_null", serviceEventClientConfiguration.getClientName());
+        assertNull("not_null", serviceEventClientConfiguration.getEventListener());
+    }
+
+    @Test
+    public void constructorAddressAndNameCharCheckTest() {
+        String[] permittedValues = {"", "!@#$%^^&**(-()_)+/|", "regularNaming", "regular Naming", "49", "regularNaming49", "NAMING", "246465494135646120009090049684646496468456468496846464968496844"};
```

Comment:
In second entry, there are double same symbols and some are not included. Please change to: "!"#$%&'()=?⁄@‹›€°·‚,.-;:_Èˇ¿<>«‘”’ÉØ∏{}|ÆæÒÔÓÌÏÎÅ«»Ç◊Ñˆ¯Èˇ" - try to include as much as special symbols as possible.